Makefile.am

```diff
@@ -215,6 +215,15 @@ _usnic_files = \
 
 if HAVE_VERBS
 _usnic_files += prov/usnic/src/usdf_fake_ibv.c
+
+# If the verbs or usnic providers are being built as a DL, then we
+# need to link libibverbs into the usnic provider
+if HAVE_VERBS_DL
+_usnic_libibverbs = -libverbs
+endif
+if HAVE_USNIC_DL
+_usnic_libibverbs = -libverbs
+endif
 endif
```

Comment:
I still don't think we really need either. usnic provider now needs to link against libibverbs, just like the verbs provider. So have them both add this in their configure.m4, just like how the verbs provider does it.

The linker will just ignore multiple requests to link against the same library and just link once.